docs/scripting-element-object.rst

```diff
@@ -0,0 +1,566 @@
+.. _splash-element:
+
+Element Object
+==============
+
+Element objects are created by :ref:`splash-select` and :ref:`splash-select-all`.
```

Comment:
No, they cannot. If you return node from these methods you will receive objects like this:

```js
{ type: 'node', id: '<uuid>' }
```